<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupBox</class>
 <widget class="QGroupBox" name="GroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QPushButton" name="btnfinalizar">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Finalizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btncancelar">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnfechar">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Fechar</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>741</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Itens</string>
   </property>
   <widget class="QLineEdit" name="linenunprod_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="linedescprod_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>25</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="linequant_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>25</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="linevalorUND_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>25</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="linedesconto_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>25</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineestoque_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>25</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnpesquisaprod_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>25</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btninserir_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>25</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Inserir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnalterar">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Alterar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnexcluir">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineintens">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="linevaloritens">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>240</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Produto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor Unitário</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>% Desconto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Estoque</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Itens:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Varlor:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>52</y>
      <width>721</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="linenunped">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>63</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>15</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data/Hora</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>15</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Número</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linedatahora">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>63</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>linenunprod_2</tabstop>
  <tabstop>linedescprod_2</tabstop>
  <tabstop>btnpesquisaprod_2</tabstop>
  <tabstop>linequant_2</tabstop>
  <tabstop>linevalorUND_2</tabstop>
  <tabstop>linedesconto_2</tabstop>
  <tabstop>lineestoque_2</tabstop>
  <tabstop>btninserir_2</tabstop>
  <tabstop>btnalterar</tabstop>
  <tabstop>btnexcluir</tabstop>
  <tabstop>btnfinalizar</tabstop>
  <tabstop>btncancelar</tabstop>
  <tabstop>btnfechar</tabstop>
  <tabstop>linedatahora</tabstop>
  <tabstop>linenunped</tabstop>
  <tabstop>lineintens</tabstop>
  <tabstop>linevaloritens</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
